infrastructure/lambda-generate-document-manifest.tf

```diff
@@ -0,0 +1,132 @@
+module "generate-document-manifest-alarm" {
+  source               = "./modules/lambda_alarms"
+  lambda_function_name = module.generate-document-manifest-lambda.function_name
+  lambda_timeout       = module.generate-document-manifest-lambda.timeout
+  lambda_name          = "generate_document_manifest_handler"
+  namespace            = "AWS/Lambda"
+  alarm_actions        = [module.generate-document-manifest-alarm-topic.arn]
+  ok_actions           = [module.generate-document-manifest-alarm-topic.arn]
+  depends_on           = [module.generate-document-manifest-lambda, module.generate-document-manifest-alarm-topic]
+}
+
+module "generate-document-manifest-alarm-topic" {
+  source                = "./modules/sns"
+  sns_encryption_key_id = module.sns_encryption_key.id
+  current_account_id    = data.aws_caller_identity.current.account_id
+  topic_name            = "generate-document-manifest-topic"
+  topic_protocol        = "lambda"
+  topic_endpoint        = module.generate-document-manifest-lambda.lambda_arn
+  delivery_policy = jsonencode({
+    "Version" : "2012-10-17",
+    "Statement" : [
+      {
+        "Effect" : "Allow",
+        "Principal" : {
+          "Service" : "cloudwatch.amazonaws.com"
+        },
+        "Action" : [
+          "SNS:Publish",
+        ],
+        "Condition" : {
+          "ArnLike" : {
+            "aws:SourceArn" : "arn:aws:cloudwatch:eu-west-2:${data.aws_caller_identity.current.account_id}:alarm:*"
+          }
+        }
+        "Resource" : "*"
+      }
+    ]
+  })
+
+  depends_on = [module.generate-document-manifest-lambda, module.sns_encryption_key]
+}
+
+module "generate-document-manifest-lambda" {
+  source                   = "./modules/lambda"
+  name                     = "GenerateDocumentManifest"
+  handler                  = "handlers.generate_document_manifest_handler.lambda_handler"
+  lambda_timeout           = 900
+  lambda_ephemeral_storage = 512
+  memory_size              = 512
+  iam_role_policies = [
+    module.document_reference_dynamodb_table.dynamodb_policy,
+    module.ndr-document-store.s3_object_access_policy,
+    module.lloyd_george_reference_dynamodb_table.dynamodb_policy,
```

Comment:
we don't need `module.lloyd_george_reference_dynamodb_table.dynamodb_policy` or `module.document_reference_dynamodb_table.dynamodb_policy,`